docs_src/src/pages/documentation/example_app/templates.mdx

```diff
@@ -65,6 +65,63 @@ This is Robyn introduced him to the concept of routers and views.
   />
 </div>
 
+# Jinja Integration Instructions
+
+To integrate resource loading functionality into Jinja templates similar to Flask's `url_for`, follow these steps:
+
+1. **Create a JinjaTemplate class**: Define a class that encapsulates the Jinja environment and provides a method to add global functions.
+
+    ```python
+    from jinja2 import Environment, FileSystemLoader
+
+    class JinjaTemplate:
+        def __init__(self, templates_dir):
+            self.env = Environment(loader=FileSystemLoader(templates_dir))
+            self.add_template_global(self.url_for_static, 'url_for')
+
+        def add_template_global(self, func, name):
+            if not callable(func):
+                raise TypeError("Must be callable.")
+            self.env.globals[name] = func
+
+        def url_for_static(self, endpoint: str, **kwargs) -> str:
```

Comment:
you have added an API named `url_for` but are adding documentation for `url_for_static` ?

Reply:
Sure, I will update this asap.